.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
is it github action or pypi? I am not fully aware

Reply:
GitHub Actions @auivpy